.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*.cs]
+end_of_line = lf
+indent_style = tab
+tab_width = 4
```

Comment:
@praeclarum Are there any C# formatting styles you'd like the code to follow?   If so, i can add them here now that VS/roslyn has added support for this.  

This will enable devs using VS to edit this code without havign to worry about stomping over each other.